doc/release-notes-18335.md

```diff
@@ -0,0 +1,6 @@
+RPC changes
+-----------
+- The RPC server can process a limited number of simultaneous RPC requests.
```

Comment:
DNM - we already release 18335 in previous version; release notes for it is late to provide

Reply:
Dropped it